src/test/scala/biowdl/test/TestCompareGff.scala

```diff
@@ -23,14 +23,20 @@ package biowdl.test
 
 import java.io.File
 
-import nl.biopet.utils.biowdl.multisample.MultisamplePipeline
+import nl.biopet.utils.biowdl.annotations.TestAnnotation
+import nl.biopet.utils.biowdl.{Pipeline, fixtureFile}
 
-trait TestPipeline extends MultisamplePipeline {
+trait TestCompareGff extends Pipeline with TestAnnotation {
```

Comment:
I wouldn't include `Test` in this trait's name, this isn't the case for most pipelines.

This trait shouldn't extend `TestAnnotation`, but `Annotation`, the specific annotation traits should only get extended in the tests where these are used because different tests might use different annotations.